src/javascript/app_2/App/Components/Form/button.jsx

```diff
@@ -3,6 +3,7 @@ import React     from 'react';
 
 const Button = ({
     className = '',
+    children,
```

Comment:
@zakibinary 
Plz, add the `children` props to PropTypes too.

Reply:
Done